api/src/main/java/datawave/webservice/metadata/MetadataFieldBase.java

```diff
@@ -11,6 +11,7 @@
 import java.util.List;
 import java.util.Set;
 
+// TODO if nothing other than DefaultMetadataField extends this class, remove this class and rename DefaultMetadataField to MetadataField
```

Comment:
Was this TODO left intentionally?

Reply:
Good question.  This was a change Ceria added, so I'm not sure.  I'm going to remove the comment.